src/main/controller/WorkoutController.ts

```diff
@@ -0,0 +1,82 @@
+import { Request, Response } from 'express';
+import db from '../models';
+import { Route } from '../constant/Route';
+
+export class WorkoutController {
+  getAllWorkouts = async (_req: Request, res: Response): Promise<any> => {
+    try {
+      const workouts = await db.Workout.findAll();
+      return res.status(200).send(workouts);
+    } catch (e) {
+      return res.status(400).send({ e });
+    }
+  };
+
+  getAllWorkoutsMinimal = async (
```

Comment:
How do we feel about changing to `getAllBasicWorkouts` for consistency + grammar & then we can use `getAllDetailedWorkouts` for later one!